src/lib/updating-element.ts

```diff
@@ -233,25 +244,40 @@ export abstract class UpdatingElement extends HTMLElement {
   }
 
   /**
-   * Creates a property accessor on the element prototype if one does not exist.
-   * The property setter calls the property's `hasChanged` property option
-   * or uses a strict identity check to determine whether or not to request
-   * an update.
+   * Ensures the private `_classProperties` property metadata is created.
+   * In addition to `_finalize` this is also called in `createProperty` to
+   * ensure the `@property` decorator can add property metadata.
    */
-  static createProperty(name: PropertyKey,
-                        options:
-                            PropertyDeclaration = defaultPropertyDeclaration) {
+  /** @nocollapse */
```

Comment:
I'm surprised we need nocollapse here. Is it because of `this` access? If so, seems like an unsafe optimization by Closure.

Reply:
See above, all static methods need @nocollapse currently (closure bug workaround).